examples/postgres/docker-compose.yml

```
@@ -18,7 +18,7 @@ services:
     ports:
       - 5433:5432
   keycloak:
-    image: phasetwo/phasetwo-keycloak-test-crdb:latest
+    image: quay.io/phasetwo/phasetwo-keycloak:latest
     environment:
       PROXY_ADDRESS_FORWARDING: 'true'
       KEYCLOAK_USER: admin
```

```suggestion
      KEYCLOAK_USER: admin
      KC_DB: postgres
```

Or you end up using the default `dev-file` and wondering where all your data goes when you bump your container tag :smile:

---

LOL. Whoops. Thanks for the PR